2025-05-18 23:09:58,512 - agriscouting_parse - INFO - Используется прокси: socks5://103.174.102.127:8080
2025-05-18 23:09:58,512 - agriscouting_parse - INFO - Запуск парсинга данных для AgriScouting
2025-05-18 23:09:58,822 - agriscouting_parse - ERROR - Ошибка при запросе https://www.syngenta.by/bolezni-zernovyh-kultur: SOCKSHTTPSConnectionPool(host='www.syngenta.by', port=443): Max retries exceeded with url: /bolezni-zernovyh-kultur (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSHTTPSConnection object at 0x000001BC2EDD3EC0>: Failed to establish a new connection: SOCKS5 proxy server sent invalid data'))
2025-05-18 23:10:01,119 - agriscouting_parse - ERROR - Ошибка при запросе https://www.syngenta.by/bolezni-zernovyh-kultur: SOCKSHTTPSConnectionPool(host='www.syngenta.by', port=443): Max retries exceeded with url: /bolezni-zernovyh-kultur (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSHTTPSConnection object at 0x000001BC2EEA14F0>: Failed to establish a new connection: SOCKS5 proxy server sent invalid data'))
2025-05-18 23:10:04,416 - agriscouting_parse - ERROR - Ошибка при запросе https://www.syngenta.by/bolezni-zernovyh-kultur: SOCKSHTTPSConnectionPool(host='www.syngenta.by', port=443): Max retries exceeded with url: /bolezni-zernovyh-kultur (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSHTTPSConnection object at 0x000001BC2EEA1D90>: Failed to establish a new connection: SOCKS5 proxy server sent invalid data'))
2025-05-18 23:10:08,417 - agriscouting_parse - ERROR - Не удалось получить https://www.syngenta.by/bolezni-zernovyh-kultur после 3 попыток
2025-05-18 23:10:08,714 - agriscouting_parse - ERROR - Ошибка при запросе https://betaren.ru/harmful/bolezni/: SOCKSHTTPSConnectionPool(host='betaren.ru', port=443): Max retries exceeded with url: /harmful/bolezni/ (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSHTTPSConnection object at 0x000001BC2EADEF60>: Failed to establish a new connection: SOCKS5 proxy server sent invalid data'))
